XPlat/Shopify/iOS/source/Shopify.iOS/ApiDefinition.cs

```diff
@@ -0,0 +1,1964 @@
+
+		// @property (copy, nonatomic) NSString * note;
+		[Export ("note")]
+		string Note { get; set; }
+
+		// @property (nonatomic, copy) NSArray <BUYCheckoutAttribute *> *attributes;
+		[Export ("attributes", ArgumentSemantic.Copy)]
+		CheckoutAttribute[] Attributes { get; }
+
+		// @property (readonly, nonatomic, strong) BUYOrder * order;
+		[Export ("order", ArgumentSemantic.Strong)]
+		Order Order { get; }
+
+		// @property (assign, nonatomic) BOOL partialAddresses;
+		[Export ("partialAddresses")]
+		bool PartialAddresses { get; set; }
+
+		// -(instancetype)initWithCart:(BUYCart *)cart;
+		[Export ("initWithCart:")]
+		IntPtr Constructor (Cart cart);
+
+		// -(instancetype)initWithCartToken:(NSString *)cartToken;
+		[Export ("initWithCartToken:")]
+		IntPtr Constructor (string cartToken);
+
+		// -(BOOL)hasToken;
+		[Export ("hasToken")]
+		bool HasToken { get; }
+
+		// @property (readonly, copy, nonatomic) NSNumber * orderId __attribute__((deprecated("Available on the BUYOrder object")));
+		[Export ("orderId", ArgumentSemantic.Copy)]
+		NSNumber OrderId { get; }
+
+		// @property (readonly, nonatomic, strong) NSURL * orderStatusURL __attribute__((deprecated("Available on the BUYOrder object")));
+		[Export ("orderStatusURL", ArgumentSemantic.Strong)]
+		NSUrl OrderStatusUrl { get; }
+	}
+
+	// @interface BUYCheckoutAttribute : BUYObject <BUYSerializable>
+	[BaseType (typeof(Object), Name = "BUYCheckoutAttribute")]
+	interface CheckoutAttribute : Serializable
+	{
+		// @property (nonatomic, strong, nonnull) NSString *name;
+		[Export ("name", ArgumentSemantic.Strong)]
+		string Name { get; set; }
+
+		// @property (nonatomic, strong, nonnull) NSString *value;
+		[Export ("value", ArgumentSemantic.Strong)]
+		string Vitle { get; set; }
+	}
+
+	// @interface BUYShippingRate : BUYObject <BUYSerializable>
+	[BaseType (typeof(Object), Name = "BUYShippingRate")]
+	interface ShippingRate : Serializable
+	{
+		// @property (readonly, nonatomic, strong) NSString * shippingRateIdentifier;
+		[Export ("shippingRateIdentifier", ArgumentSemantic.Strong)]
+		string ShippingRateIdentifier { get; }
+
+		// @property (readonly, nonatomic, strong) NSString * title;
+		[Export ("title", ArgumentSemantic.Strong)]
+		string Title { get; }
+
+		// @property (readonly, nonatomic, strong) NSDecimalNumber * price;
+		[Export ("price", ArgumentSemantic.Strong)]
+		NSDecimalNumber Price { get; }
+
+		// TODO: DeliveryRange might be NSDate[]
+
+		// @property (readonly, nonatomic, strong) NSArray * deliveryRange;
+		[Export ("deliveryRange", ArgumentSemantic.Strong)]
+		NSObject[] DeliveryRange { get; }
+
+
+		// @interface ApplePay (BUYShippingRate)
+
+		// +(NSArray<PKShippingMethod *> * _Nonnull)buy_convertShippingRatesToShippingMethods:(NSArray<BUYShippingRate *> * _Nonnull)rates;
+		[Static]
+		[Export ("buy_convertShippingRatesToShippingMethods:")]
+		PKShippingMethod[] ApplePayConvertShippingRatesToShippingMethods (ShippingRate[] rates);
```

Comment:
As suggestion, I would change method name to ApplePayConvertToShippingMethods, or ConvertToShippingMethods or just Convert.